data/json/mutations.json

```diff
@@ -2521,15 +2538,151 @@
+        "threshreq" : ["THRESH_BIRD"],
+        "category" : ["BIRD"],
+        "social_modifiers" : {
+            "persuade" : 30,
+            "intimidate" : -10
+        }
+    },{
+        "type" : "mutation",
+        "id" : "PLUMAGE",
+        "name" : "Colorful Plumage",
+        "points" : 1,
+        "valid" : false,
+        "visibility" : 10,
+        "ugliness" : -5,
+        "description" : "Some of the feathers on your head have become longer and rainbow-coloured. They're quite pretty.",
+        "prereqs" : ["FEATHERS"],
+        "category" : ["BIRD"]
+    },{
+        "type" : "mutation",
+        "id" : "FLEET3",
+        "name" : "Sub-Sonic Sprinter",
+        "points" : 5,
+        "valid" : false,
+        "description" : "Your legs are insanely limber and fast-moving.  You move 40% faster on flat surfaces and 10% faster on uneven surfaces.",
+        "prereqs" : ["FLEET2"],
+        "threshreq" : ["THRESH_BIRD"],
+        "category" : ["BIRD"],
+        "types" : ["RUNNING"]
+    },{
+        "type" : "mutation",
+        "id" : "QUICK2",
+        "name" : "Quickerer",
+        "points" : 5,
+        "valid" : false,
+        "description" : "You're incredibly fast!  You get a 25% bonus to action points.",
+        "threshreq" : ["THRESH_BIRD"],
+        "category" : ["BIRD"],
+        "prereqs" : ["QUICK"]
+    },{
+        "type" : "mutation",
+        "id" : "TALONS_STIFF",
+        "name" : "Stiff Talons",
+        "points" : -2,
+        "valid" : true,
+        "visibility" : 4,
+        "ugliness" : 2,
+        "mixed_effect" : true,
+        "description" : "Your legs have become much thinner and harder, and your feet have changed to stiff talons. They prevent you from wearing footwear, but give extra protection. They're pretty sharp too.",
+        "types" : ["LEGS"],
+        "changes_to" : ["TALONS_FLEX"],
+        "category" : ["BIRD"],
+        "restricts_gear" : [ "FOOT_L", "FOOT_R" ],
+        "destroys_gear" : true,
+        "attacks" : {
+            "attack_text_u" : "You stab %s with a talon",
+            "attack_text_npc" : "%1$s stabs %2$s with a talon",
+            "chance" : 20,
+            "strength_damage" : { "damage_type" : "stab", "amount" : 4 }
+        }
+    },{
+        "type" : "mutation",
+        "id" : "TALONS_FLEX",
+        "name" : "Flexible Talons",
+        "points" : 2,
+        "visibility" : 4,
+        "ugliness" : 2,
+        "mixed_effect" : false,
+        "description" : "The long talons on your feet have become shorter, and much more flexible, enabling you to wear shoes again.",
+        "valid" : false,
+        "types" : ["LEGS"],
+        "threshreq" : ["THRESH_BIRD"],
+        "category" : ["BIRD"],
+        "attacks" : {
+            "attack_text_u" : "You stab %s with a talon",
+            "attack_text_npc" : "%1$s stabs %2$s with a talon",
+            "chance" : 20,
+            "strength_damage" : { "damage_type" : "stab", "amount" : 2 }
```

Comment:
It doesn't make much sense to stab things with your talons if you're wearing shoes, either.